operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -333,16 +328,47 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
     }
 
     /**
-     * @return the list of patient subresources that will be included in the $everything operaetion
+     * @return the list of patient subresources that will be included in the $everything operation
      * @throws FHIRSearchException
      */
     private List<String> getDefaultIncludedResourceTypes() throws FHIRSearchException {
         List<String> resourceTypes = new ArrayList<>(CompartmentUtil.getCompartmentResourceTypes(PATIENT));
-        // TODO: Practitioner and Organization are not included in the getCompartmentReourceTypes() by default but it seems
-        // like a couple of good additional resources to include and they are even mentioned as examples of resources
-        // to include in the docs: https://www.hl7.org/fhir/operation-patient-everything.html
-        // resourceTypes.add(Practitioner.class.getSimpleName());
-        // resourceTypes.add(Organization.class.getSimpleName());
+        
+        try {
+            List<String> supportedResourceTypes = FHIRConfigHelper.getSupportedResourceTypes();
+            if (LOG.isLoggable(Level.FINE)) {
+                StringBuilder resourceTypeBuilder = new StringBuilder(supportedResourceTypes.size());
+                resourceTypeBuilder.append("supportedResourceTypes are: ");
+                for (String resourceType: supportedResourceTypes) {
+                   resourceTypeBuilder.append(resourceType);
+                   resourceTypeBuilder.append(" ");
+                }
+                LOG.fine(resourceTypeBuilder.toString());
+            }
+            // TODO: Practitioner and Organization are not included in the getCompartmentReourceTypes() by default but it seems
+            // like a couple of good additional resources to include and they are even mentioned as examples of resources
+            // to include in the docs: https://www.hl7.org/fhir/operation-patient-everything.html
+            // resourceTypes.add(Practitioner.class.getSimpleName());
+            // resourceTypes.add(Organization.class.getSimpleName());
```

Comment:
```suggestion
            // TODO: Practitioner and Organization are not included in the getCompartmentReourceTypes() by default but it seems
            // like a couple of good additional resources to include and they are even mentioned as examples of resources
            // to include in the docs: https://www.hl7.org/fhir/operation-patient-everything.html
            // resourceTypes.add(Practitioner.class.getSimpleName());
            // resourceTypes.add(Organization.class.getSimpleName());
```

Maybe this can be dropped now? or point to configuration. 

Please update the conformance document - [Conformance.md](https://github.com/IBM/FHIR/blob/main/docs/src/pages/Conformance.md#type-operations)  
1. Add an Entry under instance (the next section after this link)
2. Point to the configuration in 'open' 
3. Mention that only supported resources will be included

Should be quick blurbs...

Reply:
Added in requested documentation.